Patches/OpenCV/3.1.0/OpenCVDetectCXXCompiler.cmake

```diff
@@ -168,6 +168,8 @@ if(MSVC)
     set(OpenCV_RUNTIME vc15)
   elseif(MSVC_VERSION EQUAL 1911)
     set(OpenCV_RUNTIME vc15)
+  elseif(MSVC_VERSION EQUAL 1912)
```

Comment:
This line is ok, but it would probably be better to do a greater_equal 1900 or similar. My fear is we patch this line for every minor iteration of MSVC